services/src/main/java/org/keycloak/services/resources/KeycloakApplication.java

```diff
@@ -182,7 +182,6 @@ protected void shutdown() {
     protected ExportImportManager migrateAndBootstrap() {
         ExportImportManager exportImportManager;
         logger.debug("Calling migrateModel");
-        migrateModel();
```

Comment:
Why move this?

Reply:
And I can ask, why not have model migration together with persistence bootstrap? Why have part in JPA layer and other bits outside. It just better, IMO, to have them where persistence is initialized.